internal/builtinfunctions/functions.go

```diff
@@ -529,6 +532,37 @@ func getReadFileFunction() schema.CallableFunction {
 	return funcSchema
 }
 
+func getReadEnvVarFunction() schema.CallableFunction {
+	funcSchema, err := schema.NewCallableFunction(
+		"readEnvVar",
+		[]schema.Type{
+			schema.NewStringSchema(nil, nil, nil), // env var key name
+			schema.NewStringSchema(nil, nil, nil), // default value
+		},
+		schema.NewStringSchema(nil, nil, nil),
+		false,
+		schema.NewDisplayValue(
+			schema.PointerTo("readFile"),
```

Comment:
Maybe, `readEnvVar`? Although, in this context, I wonder about the choice of the name "read" in the first place. Maybe `getEnv` (or, if you insist, `getEnvVar`) would be better?

Reply:
Right, this is a cut-and-paste-o from line 511....

Reply:
Fixed, and changed the name.